### PR TITLE
Handle EDB and Keycloak profile sizes

### PR DESCRIPTION
**Pull request by pgodowski**
Addresses the EDB profiles as requested in https://github.ibm.com/IBMPrivateCloud/roadmap/issues/60915

### CODE REVIEW

File: controllers/size/large_amd64.go [hunk 1]
```diff
@@ -674,4 +674,15 @@ const Large = `
...
     apicatalogmanager:
       profile: large
+- name: edb-keycloak
+  spec:
+    Cluster:
+      instances: 3
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/large_amd64.go:684]:
```suggestion
          cpu: 750m
          memory: 1500Mi
```

File: controllers/size/large_amd64.go [hunk 2]
```diff
@@ -674,4 +674,15 @@ const Large = `
...
+        limits:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/large_amd64.go:687]:
```suggestion
          cpu: 750m
          memory: 1500Mi
```

File: controllers/size/large_ppc64le.go [hunk 3]
```diff
@@ -674,4 +674,15 @@ const Large = `
...
     apicatalogmanager:
       profile: large
+- name: edb-keycloak
+  spec:
+    Cluster:
+      instances: 3
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/large_ppc64le.go:684]:
```suggestion
          cpu: 750m
          memory: 1500Mi
```

File: controllers/size/large_ppc64le.go [hunk 4]
```diff
@@ -674,4 +674,15 @@ const Large = `
...
+        limits:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/large_ppc64le.go:687]:
```suggestion
          cpu: 750m
          memory: 1500Mi
```

File: controllers/size/large_s390x.go [hunk 5]
```diff
@@ -674,4 +674,15 @@ const Large = `
...
     apicatalogmanager:
       profile: large
+- name: edb-keycloak
+  spec:
+    Cluster:
+      instances: 3
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/large_s390x.go:684]:
```suggestion
          cpu: 750m
          memory: 1500Mi
```

File: controllers/size/large_s390x.go [hunk 6]
```diff
@@ -674,4 +674,15 @@ const Large = `
...
+        limits:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/large_s390x.go:687]:
```suggestion
          cpu: 750m
          memory: 1500Mi
```

File: controllers/size/medium_amd64.go [hunk 7]
```diff
@@ -674,4 +674,15 @@ const Medium = `
...
     apicatalogmanager:
       profile: medium
+- name: edb-keycloak
+  spec:
+    Cluster:
+      instances: 2
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/medium_amd64.go:684]:
```suggestion
          cpu: 500m
          memory: 1024Mi
```

File: controllers/size/medium_amd64.go [hunk 8]
```diff
@@ -674,4 +674,15 @@ const Medium = `
...
+        limits:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/medium_amd64.go:687]:
```suggestion
          cpu: 500m
          memory: 1024Mi
```

File: controllers/size/small_amd64.go [hunk 9]
```diff
@@ -674,4 +674,15 @@ const Small = `
...
     apicatalogmanager:
       profile: small
+- name: edb-keycloak
+  spec:
+    Cluster:
+      instances: 1
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/small_amd64.go:684]:
```suggestion
          cpu: 200m
          memory: 768Mi
```

File: controllers/size/small_ppc64le.go [hunk 10]
```diff
@@ -674,4 +674,15 @@ const Small = `
...
+  spec:
+    Cluster:
+      instances: 1
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
+        limits:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/small_ppc64le.go:687]:
```suggestion
          cpu: 200m
          memory: 768Mi
```

File: controllers/size/small_s390x.go [hunk 11]
```diff
@@ -674,4 +674,15 @@ const Small = `
...
     apicatalogmanager:
       profile: small
+- name: edb-keycloak
+  spec:
+    Cluster:
+      instances: 1
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/small_s390x.go:684]:
```suggestion
          cpu: 200m
          memory: 768Mi
```

File: controllers/size/small_s390x.go [hunk 12]
```diff
@@ -674,4 +674,15 @@ const Small = `
...
+        limits:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/small_s390x.go:687]:
```suggestion
          cpu: 200m
          memory: 768Mi
```

File: controllers/size/starterset_amd64.go [hunk 13]
```diff
@@ -674,4 +674,15 @@ const StarterSet = `
...
     apicatalogmanager:
       profile: small
+- name: edb-keycloak
+  spec:
+    Cluster:
+      instances: 1
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/starterset_amd64.go:684]:
```suggestion
          cpu: 200m
          memory: 512Mi
```

File: controllers/size/starterset_amd64.go [hunk 14]
```diff
@@ -674,4 +674,15 @@ const StarterSet = `
...
+        limits:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/starterset_amd64.go:687]:
```suggestion
          cpu: 200m
          memory: 512Mi
```

File: controllers/size/starterset_ppc64le.go [hunk 15]
```diff
@@ -674,4 +674,15 @@ const StarterSet = `
...
     apicatalogmanager:
       profile: small
+- name: edb-keycloak
+  spec:
+    Cluster:
+      instances: 1
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/starterset_ppc64le.go:684]:
```suggestion
          cpu: 200m
          memory: 512Mi
```

File: controllers/size/starterset_ppc64le.go [hunk 16]
```diff
@@ -674,4 +674,15 @@ const StarterSet = `
...
+        limits:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/starterset_ppc64le.go:687]:
```suggestion
          cpu: 200m
          memory: 512Mi
```

File: controllers/size/starterset_s390x.go [hunk 17]
```diff
@@ -676,4 +676,15 @@ const StarterSet = `
...
     apicatalogmanager:
       profile: small
+- name: edb-keycloak
+  spec:
+    Cluster:
+      instances: 1
+      resources:
+        requests:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/starterset_s390x.go:686]:
```suggestion
          cpu: 200m
          memory: 512Mi
```

File: controllers/size/starterset_s390x.go [hunk 18]
```diff
@@ -676,4 +676,15 @@ const StarterSet = `
...
+        limits:
+          cpu: 1000m
+          memory: 1Gi
```
Comment (giacomoch) [inline controllers/size/starterset_s390x.go:689]:
```suggestion
          cpu: 200m
          memory: 512Mi
```

**Comment by Daniel-Fan** [comment]
PP is opened to unblock the EDB and KeyCloak profile sizing https://github.com/IBM/ibm-common-service-operator/pull/1585

@pgodowski 
Could you please kindly review @YCShen1010 's PR against your branch? https://github.com/pgodowski/ibm-common-service-operator/pull/1

**Comment by pgodowski** [comment]
Thanks, merged the change from @YCShen1010

**Review by Daniel-Fan — APPROVED** [review]
/lgtm

**Pull request merged**